### PR TITLE
HBASE-25189 [Metrics] Add checkAndPut and checkAndDelete latency metrics at table level

### PR DESCRIPTION
**Pull request by Reidddddd**
[HBASE-25189](https://issues.apache.org/jira/browse/HBASE-25189)

### CODE REVIEW

File: hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestMetricsRegionServer.java [hunk 1]
```diff
@@ -162,17 +162,6 @@ public void testSlowCount() {
     HELPER.assertCounter("slowPutCount", 16, serverSource);
   }
 
-  String FLUSH_TIME = "flushTime";
```
Comment (Reidddddd) [inline hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestMetricsRegionServer.java:165]: they are unused, that's why i removed them.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   2m 28s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ branch-1 Compile Tests _ |
| +0 :ok: |  mvndep  |   2m 29s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   9m 42s |  branch-1 passed  |
| +1 :green_heart: |  compile  |   1m 42s |  branch-1 passed with JDK v1.8.0_262  |
| +1 :green_heart: |  compile  |   2m  0s |  branch-1 passed with JDK v1.7.0_272  |
| +1 :green_heart: |  checkstyle  |   3m 15s |  branch-1 passed  |
| +1 :green_heart: |  shadedjars  |   4m 45s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 40s |  branch-1 passed with JDK v1.8.0_262  |
| +1 :green_heart: |  javadoc  |   1m 56s |  branch-1 passed with JDK v1.7.0_272  |
| +0 :ok: |  spotbugs  |   4m 48s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   6m 45s |  branch-1 passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 24s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 27s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 49s |  the patch passed with JDK v1.8.0_262  |
| +1 :green_heart: |  javac  |   1m 49s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 59s |  the patch passed with JDK v1.7.0_272  |
| +1 :green_heart: |  javac  |   1m 59s |  the patch passed  |
| -1 :x: |  checkstyle  |   2m 30s |  hbase-server: The patch generated 2 new + 156 unchanged - 2 fixed = 158 total (was 158)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedjars  |   4m 39s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |   7m 59s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2.  |
| +1 :green_heart: |  javadoc  |   1m 40s |  the patch passed with JDK v1.8.0_262  |
| +1 :green_heart: |  javadoc  |   1m 52s |  the patch passed with JDK v1.7.0_272  |
| +1 :green_heart: |  findbugs  |   7m 26s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 40s |  hbase-hadoop-compat in the patch passed.  |
| +1 :green_heart: |  unit  |   0m 56s |  hbase-hadoop2-compat in the patch passed.  |
| -1 :x: |  unit  | 213m 35s |  hbase-server in the patch failed.  |
| +1 :green_heart: |  asflicense  |   2m  1s |  The patch does not generate ASF License warnings.  |
|  |   | 295m 15s |   |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.client.TestIncrementFromClientSideWithCoprocessor |
|   | hadoop.hbase.mapreduce.TestImportExport |
|   | hadoop.hbase.replication.TestReplicationKillMasterRSCompressed |
|   | hadoop.hbase.replication.TestMultiSlaveReplication |
|   | hadoop.hbase.client.TestSplitOrMergeStatusWithZKBasedAssignment |
|   | hadoop.hbase.replication.TestReplicationWithTags |
|   | hadoop.hbase.coprocessor.TestWALObserver |
|   | hadoop.hbase.coprocessor.TestRegionObserverForAddingMutationsFromCoprocessors |
|   | hadoop.hbase.client.TestIncrementsFromClientSide |
|   | hadoop.hbase.coprocessor.TestRegionCoprocessorHost |
|   | hadoop.hbase.client.TestUpdateConfiguration |
|   | hadoop.hbase.replication.TestReplicationChangingPeerRegionservers |
|   | hadoop.hbase.coprocessor.TestCoprocessorStop |
|   | hadoop.hbase.regionserver.TestRemoveRegionMetrics |
|   | hadoop.hbase.replication.multiwal.TestReplicationKillMasterRSCompressedWithMultipleWAL |
|   | hadoop.hbase.replication.TestPerTableCFReplication |
|   | hadoop.hbase.replication.regionserver.TestGlobalReplicationThrottler |
|   | hadoop.hbase.coprocessor.TestMasterCoprocessorExceptionWithAbort |
|   | hadoop.hbase.client.TestSplitOrMergeStatus |
|   | hadoop.hbase.coprocessor.TestNegativeMemstoreSizeWithSlowCoprocessor |
|   | hadoop.hbase.replication.TestReplicationConfigTracker |
|   | hadoop.hbase.regionserver.wal.TestLogRolling |
|   | hadoop.hbase.coprocessor.TestDoubleColumnInterpreter |
|   | hadoop.hbase.regionserver.wal.TestWALReplayBoundedLogWriterCreation |
|   | hadoop.hbase.replication.regionserver.TestReplicationSink |
|   | hadoop.hbase.mapreduce.TestSyncTable |
|   | hadoop.hbase.TestMetaTableAccessor |
|   | hadoop.hbase.regionserver.TestRegionServerOnlineConfigChange |
|   | hadoop.hbase.regionserver.wal.TestWALReplayCompressed |
|   | hadoop.hbase.client.TestTimestampsFilter |
|   | hadoop.hbase.client.TestClientOperationInterrupt |
|   | hadoop.hbase.replication.TestReplicationMetricsforUI |
|   | hadoop.hbase.replication.TestReplicationStatus |
|   | hadoop.hbase.replication.TestReplicationSmallTests |
|   | hadoop.hbase.client.TestScannerTimeout |
|   | hadoop.hbase.client.TestMetaScanner |
|   | hadoop.hbase.client.TestPutWithDelete |
|   | hadoop.hbase.coprocessor.TestRegionObserverInterface |
|   | hadoop.hbase.regionserver.TestCompoundBloomFilter |
|   | hadoop.hbase.client.replication.TestReplicationAdminWithTwoDifferentZKClusters |
|   | hadoop.hbase.replication.TestReplicationSyncUpToolWithBulkLoadedData |
|   | hadoop.hbase.replication.TestReplicationEndpoint |
|   | hadoop.hbase.coprocessor.TestRowProcessorEndpoint |
|   | hadoop.hbase.coprocessor.TestBatchCoprocessorEndpoint |
|   | hadoop.hbase.coprocessor.TestRegionObserverScannerOpenHook |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.13 Server=19.03.13 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2548 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux ce2f1a633388 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-home/workspace/Base-PreCommit-GitHub-PR_PR-2548/out/precommit/personality/provided.sh |
| git revision | branch-1 / e4a8dcc |
| Default Java | 1.7.0_272 |
| Multi-JDK versions | /usr/lib/jvm/zulu-8-amd64:1.8.0_262 /usr/lib/jvm/zulu-7-amd64:1.7.0_272 |
| checkstyle | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/1/artifact/out/diff-checkstyle-hbase-server.txt |
| unit | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/1/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/1/testReport/ |
| Max. process+thread count | 2861 (vs. ulimit of 10000) |
| modules | C: hbase-hadoop-compat hbase-hadoop2-compat hbase-server U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/1/console |
| versions | git=1.9.1 maven=3.0.5 findbugs=3.0.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 17s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ branch-1 Compile Tests _ |
| +0 :ok: |  mvndep  |   2m 27s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   8m 15s |  branch-1 passed  |
| +1 :green_heart: |  compile  |   1m 16s |  branch-1 passed with JDK v1.8.0_262  |
| +1 :green_heart: |  compile  |   1m 25s |  branch-1 passed with JDK v1.7.0_272  |
| +1 :green_heart: |  checkstyle  |   2m 15s |  branch-1 passed  |
| +1 :green_heart: |  shadedjars  |   3m 13s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 14s |  branch-1 passed with JDK v1.8.0_262  |
| +1 :green_heart: |  javadoc  |   1m 19s |  branch-1 passed with JDK v1.7.0_272  |
| +0 :ok: |  spotbugs  |   3m  2s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   4m 30s |  branch-1 passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 17s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   2m  4s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 14s |  the patch passed with JDK v1.8.0_262  |
| +1 :green_heart: |  javac  |   1m 14s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 23s |  the patch passed with JDK v1.7.0_272  |
| +1 :green_heart: |  javac  |   1m 23s |  the patch passed  |
| -1 :x: |  checkstyle  |   1m 35s |  hbase-server: The patch generated 2 new + 156 unchanged - 2 fixed = 158 total (was 158)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedjars  |   2m 59s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |   4m 52s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2.  |
| +1 :green_heart: |  javadoc  |   1m  5s |  the patch passed with JDK v1.8.0_262  |
| +1 :green_heart: |  javadoc  |   1m 21s |  the patch passed with JDK v1.7.0_272  |
| +1 :green_heart: |  findbugs  |   4m 36s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 29s |  hbase-hadoop-compat in the patch passed.  |
| +1 :green_heart: |  unit  |   0m 41s |  hbase-hadoop2-compat in the patch passed.  |
| +1 :green_heart: |  unit  | 120m  8s |  hbase-server in the patch passed.  |
| +1 :green_heart: |  asflicense  |   1m 19s |  The patch does not generate ASF License warnings.  |
|  |   | 176m 16s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.13 Server=19.03.13 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2548 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 859e0bd88aca 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-home/workspace/Base-PreCommit-GitHub-PR_PR-2548/out/precommit/personality/provided.sh |
| git revision | branch-1 / e066951 |
| Default Java | 1.7.0_272 |
| Multi-JDK versions | /usr/lib/jvm/zulu-8-amd64:1.8.0_262 /usr/lib/jvm/zulu-7-amd64:1.7.0_272 |
| checkstyle | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/2/artifact/out/diff-checkstyle-hbase-server.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/2/testReport/ |
| Max. process+thread count | 3747 (vs. ulimit of 10000) |
| modules | C: hbase-hadoop-compat hbase-hadoop2-compat hbase-server U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/2/console |
| versions | git=1.9.1 maven=3.0.5 findbugs=3.0.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  7s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ branch-1 Compile Tests _ |
| +0 :ok: |  mvndep  |   2m 27s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   8m 18s |  branch-1 passed  |
| +1 :green_heart: |  compile  |   1m 20s |  branch-1 passed with JDK v1.8.0_262  |
| +1 :green_heart: |  compile  |   1m 23s |  branch-1 passed with JDK v1.7.0_272  |
| +1 :green_heart: |  checkstyle  |   2m 14s |  branch-1 passed  |
| +1 :green_heart: |  shadedjars  |   3m 10s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 14s |  branch-1 passed with JDK v1.8.0_262  |
| +1 :green_heart: |  javadoc  |   1m 21s |  branch-1 passed with JDK v1.7.0_272  |
| +0 :ok: |  spotbugs  |   2m 51s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   4m 18s |  branch-1 passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 19s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   2m  2s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 14s |  the patch passed with JDK v1.8.0_262  |
| +1 :green_heart: |  javac  |   1m 14s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 24s |  the patch passed with JDK v1.7.0_272  |
| +1 :green_heart: |  javac  |   1m 24s |  the patch passed  |
| -1 :x: |  checkstyle  |   1m 35s |  hbase-server: The patch generated 6 new + 156 unchanged - 2 fixed = 162 total (was 158)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedjars  |   3m  3s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |   4m 52s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2.  |
| +1 :green_heart: |  javadoc  |   1m  3s |  the patch passed with JDK v1.8.0_262  |
| +1 :green_heart: |  javadoc  |   1m 21s |  the patch passed with JDK v1.7.0_272  |
| +1 :green_heart: |  findbugs  |   4m 39s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 30s |  hbase-hadoop-compat in the patch passed.  |
| +1 :green_heart: |  unit  |   0m 41s |  hbase-hadoop2-compat in the patch passed.  |
| -1 :x: |  unit  | 120m 27s |  hbase-server in the patch failed.  |
| +1 :green_heart: |  asflicense  |   1m 17s |  The patch does not generate ASF License warnings.  |
|  |   | 176m 12s |   |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.client.TestMetaCache |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.13 Server=19.03.13 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2548 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux dc72273419a9 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-home/workspace/Base-PreCommit-GitHub-PR_PR-2548/out/precommit/personality/provided.sh |
| git revision | branch-1 / e066951 |
| Default Java | 1.7.0_272 |
| Multi-JDK versions | /usr/lib/jvm/zulu-8-amd64:1.8.0_262 /usr/lib/jvm/zulu-7-amd64:1.7.0_272 |
| checkstyle | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/3/artifact/out/diff-checkstyle-hbase-server.txt |
| unit | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/3/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/3/testReport/ |
| Max. process+thread count | 3475 (vs. ulimit of 10000) |
| modules | C: hbase-hadoop-compat hbase-hadoop2-compat hbase-server U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/3/console |
| versions | git=1.9.1 maven=3.0.5 findbugs=3.0.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |  13m  3s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ branch-1 Compile Tests _ |
| +0 :ok: |  mvndep  |   2m 23s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   8m  0s |  branch-1 passed  |
| +1 :green_heart: |  compile  |   1m 12s |  branch-1 passed with JDK v1.8.0_262  |
| +1 :green_heart: |  compile  |   1m 17s |  branch-1 passed with JDK v1.7.0_272  |
| +1 :green_heart: |  checkstyle  |   2m  7s |  branch-1 passed  |
| +1 :green_heart: |  shadedjars  |   3m  5s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 10s |  branch-1 passed with JDK v1.8.0_262  |
| +1 :green_heart: |  javadoc  |   1m 17s |  branch-1 passed with JDK v1.7.0_272  |
| +0 :ok: |  spotbugs  |   2m 42s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   4m  5s |  branch-1 passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 18s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   1m 56s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 10s |  the patch passed with JDK v1.8.0_262  |
| +1 :green_heart: |  javac  |   1m 10s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 20s |  the patch passed with JDK v1.7.0_272  |
| +1 :green_heart: |  javac  |   1m 20s |  the patch passed  |
| -1 :x: |  checkstyle  |   1m 32s |  hbase-server: The patch generated 2 new + 156 unchanged - 2 fixed = 158 total (was 158)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedjars  |   2m 50s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |   4m 42s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2.  |
| +1 :green_heart: |  javadoc  |   0m 58s |  the patch passed with JDK v1.8.0_262  |
| +1 :green_heart: |  javadoc  |   1m 15s |  the patch passed with JDK v1.7.0_272  |
| +1 :green_heart: |  findbugs  |   4m 25s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 30s |  hbase-hadoop-compat in the patch passed.  |
| +1 :green_heart: |  unit  |   0m 39s |  hbase-hadoop2-compat in the patch passed.  |
| +1 :green_heart: |  unit  | 119m 20s |  hbase-server in the patch passed.  |
| +1 :green_heart: |  asflicense  |   1m 15s |  The patch does not generate ASF License warnings.  |
|  |   | 184m 30s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.13 Server=19.03.13 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2548 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux c9d2e763088f 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-home/workspace/Base-PreCommit-GitHub-PR_PR-2548/out/precommit/personality/provided.sh |
| git revision | branch-1 / 9eb3a79 |
| Default Java | 1.7.0_272 |
| Multi-JDK versions | /usr/lib/jvm/zulu-8-amd64:1.8.0_262 /usr/lib/jvm/zulu-7-amd64:1.7.0_272 |
| checkstyle | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/4/artifact/out/diff-checkstyle-hbase-server.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/4/testReport/ |
| Max. process+thread count | 3456 (vs. ulimit of 10000) |
| modules | C: hbase-hadoop-compat hbase-hadoop2-compat hbase-server U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2548/4/console |
| versions | git=1.9.1 maven=3.0.5 findbugs=3.0.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Reidddddd** [comment]
It is exactly the same as other codes
```
          if (request.hasCondition()) {
            regionServer.metricsRegionServer.updateCheckAndDelete(
                region == null ? null : region.getRegionInfo().getTable(), after - before);
          } else {
            regionServer.metricsRegionServer.updateDelete(
                region == null ? null : region.getRegionInfo().getTable(), after - before);
          }
```
I'm not going to spend too much efforts on it.

**Pull request merged**